Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/Functions/log.materialgraphnode

```diff
@@ -5,8 +5,9 @@
     "ClassData": {
         "id": "{A597FC44-043F-42A0-8CB6-FF43A5A889DC}",
         "category": "Math Functions",
-        "title": "Log",
+        "title": "Log Base E",
```

Comment:
Isn't this usually termed the natural log (Ln)

Reply:
Yes. But then it would be sorted differently. All of the display names, titles, categories, descriptions, etc. are purely for display and can be updated at anytime if someone wants to add detail or correct something.